test/e2e/service_options_test.go

```diff
@@ -85,6 +85,13 @@ func TestServiceOptions(t *testing.T) {
 	validateServiceAnnotations(t, it, r, "svc3", map[string]string{"alpha": "direwolf", "brave": ""})
 	serviceDelete(t, it, r, "svc3")
 
+	t.Log("create, update and validate service with annotations but -a")
+	serviceCreateWithOptions(t, it, r, "svc3", "-a", "alpha=wolf", "-a", "brave=horse")
+	validateServiceAnnotations(t, it, r, "svc3", map[string]string{"alpha": "wolf", "brave": "horse"})
+	serviceUpdate(t, it, r, "svc3", "-a", "alpha=direwolf", "-a", "brave-")
+	validateServiceAnnotations(t, it, r, "svc3", map[string]string{"alpha": "direwolf", "brave": ""})
+	serviceDelete(t, it, r, "svc3")
+
```

Comment:
You probably have to adapt this part to fit the refactoring landed recently (that's why the build tests are failing)